### PR TITLE
Alphabetical order

### PR DESCRIPTION
**Pull request by vxav**
### What this PR does / why we need it

- Fix alphabetical order of sub articles
- Small changes to vsphere/vcd
- Change section names to work better with on-prem providers

### Things to check/remember before submitting

- If you made content changes

    - Run `make lint dev` to render and proofread content changes locally.
    - Bump `last_review_date` in the front matter header if you reviewed the entire page.

### CODE REVIEW

File: src/content/getting-started/prepare-your-provider-account/vmware-vsphere/_index.md [hunk 1]
```diff
@@ -72,10 +74,9 @@ The `controller` in NSX ALB plays a pivotal role. It's responsible for communica
 
 The Cluster API controller that provisions the infrastructure in the vSphere environment needs a role with a set of permissions. To follow the principle of least privilege, it's recommended that a specific user and role be created for the controller.
 
-> [!CAUTION]
-> The password mustn’t contain ` \ ` (backslash) characters. Ideally restrict special characters to ` . , ! ? - `
+**Warning**: The password mustn’t contain ` \ ` (backslash) characters. Ideally restrict special characters to ` . , ! ? - `
```
Comment (AndiDog) [inline src/content/getting-started/prepare-your-provider-account/vmware-vsphere/_index.md:77]:
```suggestion
**Warning**: The password may not contain `\` (backslash) characters. Ideally restrict special characters to ` . , ! ? - `
```

("must not" isn't commonly known in non-natives)

**Review by AndiDog — COMMENTED** [review]
Fine, just one minor remark

**Review by AndiDog — APPROVED** [review]

**Comment by marians** [comment]
You changed the titles, but in the URL, it's still `prepare-your-provider-account`. Should we adapt that, too?

**Review by pipo02mix — APPROVED** [review]
noice

**Comment by vxav** [comment]
@marians rename the folder ? 
https://github.com/giantswarm/docs/pull/2350/commits/b2ec0699a06672392424e4aae4fc1e46bb8e32c3

**Review by marians — APPROVED** [review]
Thank you!

**Pull request merged**